bedrock/firefox/redirects.py

```diff
@@ -629,4 +629,11 @@ def firefox_channel(*args, **kwargs):
 
     # issue 7670
     redirect(r'^/firefox/fights-for-you/?', 'firefox'),
+
+    # issue #7424
+    redirect(r'^/firefox/content-blocking/start/', 'https://support.mozilla.org/'),
```

Comment:
Please remove the URL definitions for the paths you're redirecting here:

https://github.com/mozilla/bedrock/blob/master/bedrock/firefox/urls.py#L77-L80
https://github.com/mozilla/bedrock/blob/master/bedrock/firefox/urls.py#L17-L18

You'll also need to update these redirects to support the URLs when they include a version number. For example:

- /firefox/69.0/tracking-protection/start/
- /firefox/69.0.1/tracking-protection/start/
- /firefox/69.0a2/tracking-protection/start/
- /firefox/69.0a1/tracking-protection/start/

You'll likely need to use a regex to accomplish this. Adding some tests for the redirects will probably be needed here.

Let me know if you need any help with this part.

Reply:
The regex here may be very similar to what you need:

https://github.com/mozilla/bedrock/blob/master/bedrock/firefox/redirects.py#L522

It may be a case of copying it, and then changing the end to match `/content-blocking/start/` and `/tracking-protection/start/`

Reply:
@alexgibson  Can you plz help me with Regex for this?